app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/view/DataBrokerView.kt

```diff
@@ -149,64 +153,108 @@ private fun TopBar(
             containerColor = MaterialTheme.colorScheme.primaryContainer,
         ),
         actions = {
-            OverflowMenu {
-                Row(
-                    modifier = Modifier
-                        .clickable(
-                            enabled = connectionViewModel.isDisconnected,
-                        ) {
-                            val newValue = !topAppBarViewModel.isCompatibilityModeEnabled
-                            topAppBarViewModel.isCompatibilityModeEnabled = newValue
-                        }
-                        .padding(horizontal = 16.dp),
+            ConnectionStatusIcon(connectionViewModel)
+            HamburgerMenu(connectionViewModel, topAppBarViewModel)
+        },
+    )
+}
+
+@Composable
+private fun HamburgerMenu(
```

Comment:
I think a HamburgerMenu is more like 3 horizontal lines. We just got 3 dots. Maybe just "SettingsMenu" or "ModeMenu"?